frontend/src/components/sessions.js

```diff
@@ -128,6 +140,16 @@ export function ConnectedSessionsList(props) {
                 show={!!sessionToDelete}
                 onHide={() => setSessionToDelete(null)}
                 onDelete={async () => {
+                    if (sessionToDelete === activeSession) {
+                        await Promise.all([
```

Comment:
We have an init session that should be used instead. Look into `initFromStage`. You shouldn't be directly dispatching these fetch successes from UI code.